### DISCUSSION TITLE
Filter record by value in json field

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2255] [Q&A] [answered]
Is that possible to filter records by value in json field. For an example if a table have skills as field with following values
{
"go":5,
"php":2
}

is that possible to get records with go above or equal 5 as well as records with go skills alone.

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand your search requirements, but you can apply basic conditions on `json` field props using dot-notation, aka. `yourJsonField.go >= 5`.

If you are building the query manually, you can use the built-in SQLite [json functions](https://www.sqlite.org/json1.html).

**Reply by iamdineshkumar** [reply]
is that possible to use json functions through dart sdk? Since there are some use case for json_insert & similar functions.

**Reply by ganigeorgiev** [reply]
The SDKs (and the web APIs in general) don't support executing raw SQL functions or expressions.

**Comment by asaf** [comment]
I think this only work if `yourJsonField` is an object, in case of an array its not working

**Reply by ganigeorgiev** [reply]
For array values you need to specify the exact index you want to filter, eg. `yourJsonField.0.go > 5`. There is no wildcard support for the json filter if that's what you are looking for.
